makefile

```diff
@@ -91,29 +91,29 @@ start_sentinel: ${BINARY}
 	done
 
 stop_cluster:
-	$(call kill-redis,${CLUSTER_PID_PATHS})
-	rm -f appendonly.aof || true
-	rm -f ${CLUSTER_CONF_PATHS} || true
+	@$(call kill-redis,${CLUSTER_PID_PATHS})
+	@rm -f appendonly.aof || true
+	@rm -f ${CLUSTER_CONF_PATHS} || true
 
 start_cluster: ${BINARY}
-	for port in ${CLUSTER_PORTS}; do                    \
-		${BINARY}                                         \
-			--daemonize            yes                      \
-			--appendonly           yes                      \
-			--cluster-enabled      yes                      \
-			--cluster-config-file  ${TMP}/nodes$$port.conf  \
-			--cluster-node-timeout 5000                     \
-			--pidfile              ${TMP}/redis$$port.pid   \
-			--port                 $$port                   \
-			--unixsocket           ${TMP}/redis$$port.sock; \
+	@for port in ${CLUSTER_PORTS}; do\
+		${BINARY}\
+			--daemonize            yes\
+			--appendonly           yes\
+			--cluster-enabled      yes\
+			--cluster-config-file  ${TMP}/nodes$$port.conf\
+			--cluster-node-timeout 5000\
+			--pidfile              ${TMP}/redis$$port.pid\
+			--port                 $$port\
+			--unixsocket           ${TMP}/redis$$port.sock;\
 	done
+	@sleep 3
 
 create_cluster:
-	yes yes | ((${REDIS_CLIENT} --cluster create ${CLUSTER_ADDRS} --cluster-replicas 1) || \
-		(bundle exec ruby ${REDIS_TRIB} create --replicas 1 ${CLUSTER_ADDRS}))
+	@bin/cluster_creater ${CLUSTER_ADDRS}
```

Comment:
`redis-trib.rb` and `redis-cli` don't wait until the cluster is strictly ready.
The cluster test cases are executed later when we use test-unit.
This issue is exposed by randomization of minitest.
`test/support/cluster/orchestrator.rb` can wait exactly for cluster readiness.